plugins/outputs/opentsdb/opentsdb.go

```diff
@@ -29,21 +31,34 @@ var sampleConfig = `
   prefix = "my.specific.prefix."
 
   ## Telnet Mode ##
-  ## DNS name of the OpenTSDB server in telnet mode
+  ## DNS name of the OpenTSDB server
   host = "opentsdb.example.com"
 
   ## Port of the OpenTSDB server in telnet mode
   port = 4242
 
+  ## Use Http PUT API
+  useHttp = false
+
+  ## Number of data points to send to OpenTSDB in Http requests.
+  ## Not used when useHttp is false.
+  batchSize = 50
```

Comment:
This batch size seems a bit small to me, does OpenTSDB recommend only 50 points per batch?